doc/manual/manual.tex

```diff
@@ -6004,6 +6006,15 @@ \subsubsection{Tracking finite strain}
 Because the values of the compositional fields are part of what the material model gets as inputs, 
 they can easily be used for computing material model outputs such as the viscosity.  
 
+\note{In this model we present the use of multiple compositional fields for other purposes as chemical composition.
+It would have been feasible to run the same model with tracer particles that track the finite strain, as
+additionally implemented and tested in the simple shear and pure shear benchmarks mentioned in this section. 
+Both approaches have specific advantages, and for scientific computations one needs to evaluate the more suitable
+strategy. Compositional fields cover the whole domain, but contain numerical diffusion, effectively reducing the
+maximum accumulated strain. Tracers only present finite strain values at discrete position, but can be used in 
+restricted domains (and are much faster in this case). If however
```

Comment:
What do you mean by that? Something like: "but can, if this is desired, used in only a part of the domain"?